VideoLocker/src/main/java/org/edx/mobile/loader/CoursesAsyncLoader.java

```diff
@@ -34,7 +35,7 @@ public CoursesAsyncLoader(Context context, Bundle args){
     public AsyncTaskResult<List<EnrolledCoursesResponse>> loadInBackground() {
 
         Api api = new Api(getContext());
-        if (this.oauthToken == null){
+        if (Config.getInstance().getSocialSharingConfig().isEnabled() && this.oauthToken == null) {
```

Comment:
May be when social sharing is ON and Facebook token is null, they wanted to not load the courses?